osu.Game/OsuGame.cs

```diff
@@ -329,8 +329,7 @@ private void load()
                     break;
 
                 case LinkAction.OpenUserProfile:
-                    if (int.TryParse(link.Argument, out int userId))
-                        ShowUser(userId);
+                    ShowUser(link.Argument);
```

Comment:
While it works either way, I believe the ID overload should be used when it's supposed to be, rather than making all cases go for the "username" overload to eventually end up having the same target URL during request.
```suggestion
                    if (int.TryParse(link.Argument, out var userId))
                        ShowUser(userId);
                    else
                        ShowUser(link.Argument);

```